### DISCUSSION TITLE
提几个建议

### DISCUSSION THREAD

**Discussion by izhouteng** [element-plus/element-plus#8479] [Ideas]
Feature type: **`Other`**

## Details

- Enhanced component name: `popconfirm`

## 1. 目前不少组件的属性都是 小驼峰 和 横线连接 混合出现的。

例如：confirmButtonText、icon-color，两种属性命名风格同时出现。

建议统一改成横线连接 (kebab-case)

## 2. 建议把文档上右侧组件菜单名与组件名称对应。

例如：Pop Confirm 弹出确认框

由于组件名是“el-popconfirm”，应统一使用“Popconfirm”，而不应“Pop Confirm”、“Popconfirm”混合出现。

## 3. 目前 Popconfirm 组件，点完确认/取消后，需要等待200ms才会消失，造成一点卡顿的错觉，希望能去掉。（这是由于 Popover 组件默认延迟200ms关闭，或者希望能给出一个属性来控制延迟关闭时间）
其他组件库并没有这种延迟。

### Additional comments

_(empty)_

**Comment by sxzz** [comment]
1. 组件的 props 的两种命名方式都是等价的，在使用过程中可以根据你的代码风格随意选择。由于文档修改是一个繁重的任务，欢迎 PR。
2. 其他关于文档的优化，欢迎 PR。
